src/core/IronPython.Modules/fcntl.cs

```diff
@@ -53,10 +146,109 @@ public static object ioctl(CodeContext context, int fd, int cmd, [NotNone] IBuff
         throw new NotImplementedException($"ioctl: unsupported command {cmd}");
     }
 
+    #endregion
+
+
+    #region  flock
+
+    [DllImport("libc", SetLastError = true, EntryPoint = "flock")]
```

Comment:
Wonder what happens on Alpine. 😉

Reply:
Will check 🤷

Reply:
It works! However, `Mono.Unix` doesn't load, missing dependency `ld-linux-aarch64.so.1`, which makes ipy pretty unusable. Even when I installed extra package `gcompat` (GNU C Library compatibility layer for `musl`, providing `ld`), `Mono.Unix` was still failing with ` __sprintf_chk: symbol not found`.

Reply:
I guess my comment was more of a joke, but still unfortunate that `Mono.Unix` doesn't work on it. :cry: